### DISCUSSION TITLE
Separate Session Setup/Teardown Logs?

### DISCUSSION THREAD

**Discussion by andrewgrabbs** [pytest-dev/pytest-html#569] [Q&A]
Hello! I was wondering if there was a way to separate the session setup and teardown logs in another table on the html?

Right now I have a fixture scoped to session and autouse that does some setup & tear down before any tests are run; however, the logging on the "setup" side seems to get sent to the first test and the logging on the "teardown" side seems to get sent to the last test. 
```
LOGGER = logging.getLogger(__name__)

# Setup and Teardown
@pytest.fixture(autouse=True, scope='session')
def setup_teardown():
    LOGGER.info('setup')
    yield
    LOGGER.info('teardown')
```

Is there anyway to give these specific logs their own section in the HTML output? 
<img width="738" alt="Screenshot 2022-11-16 at 11 47 33 PM" src="https://user-images.githubusercontent.com/92738481/202366788-aa041a10-4721-43df-875e-b4cea7cd86fb.png">

**Comment by BeyondEvil** [comment]
That's an interesting use-case. 🤔 

I _think_ all the data we get from pytest is test (item) centric, but I could be wrong @RonnyPfannschmidt 

If it's not, you can likely add rows with that info.

Could you do a simple mock-up with what you would like/expect it to look like?

**Reply by RonnyPfannschmidt** [reply]
The data currently is item centric

Making it more granular would be a major refactoring and change
